src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext, useMemo } from 'react';
+
+import { TodoInfo } from '../TodoInfo';
+import { AppContext } from '../AppContext';
+import { FilterMode } from '../../types/FilterMode';
+
+export const TodoList: React.FC = React.memo(() => {
+  const {
+    allTodos,
```

Comment:
Consider destructuring `allTodos`, `tempTodo`, and `currentFilterMode` directly from the `useContext(AppContext)` call for better readability. 📖